src/App.css

```diff
@@ -24,16 +10,139 @@
   color: white;
 }
 
-.App-link {
-  color: #61dafb;
+.calculation-result {
+  background-color: gray;
+  grid-area: result;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding-right: 20px;
+  font-size: 50px;
+}
+
+.operation-divition,
+.operation-multiplication,
+.operation-minus,
+.operation-add,
+.operation-icual {
+  background-color: coral;
+  color: black;
+  margin: 2px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 40px;
+}
+
+.operationAC,
+.operation-change,
+.operation-percent,
+.number9,
+.number8,
+.number7,
+.number6,
+.number5,
+.number4,
+.number3,
+.number2,
+.number1,
+.number0,
+.number-dot {
+  background-color: white;
+  color: black;
+  margin: 2px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 40px;
+}
+
+.operationAC {
+  grid-area: ac;
+}
+
+.operation-change {
+  grid-area: change;
+}
+
+.operation-percent {
+  grid-area: percent;
+}
+
+.operation-divition {
+  grid-area: divition;
+}
+
+.number9 {
+  grid-area: number9;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.number8 {
+  grid-area: number8;
+}
+
+.number7 {
+  grid-area: number7;
+}
+
+.number6 {
+  grid-area: number6;
+}
+
+.number5 {
+  grid-area: number5;
+}
+
+.number4 {
+  grid-area: number4;
+}
+
+.number3 {
+  grid-area: number3;
+}
+
+.number2 {
+  grid-area: number2;
+}
+
+.number1 {
+  grid-area: number1;
+}
+
+.number0 {
+  grid-area: number0;
+}
+
+.number-dot {
+  grid-area: number-dot;
+}
+
+.operation-multiplication {
+  grid-area: multiplication;
+}
+
+.operation-minus {
+  grid-area: minus;
+}
+
+.operation-add {
+  grid-area: addition;
+}
```

Comment:
currently your design looks like this with no styling on the `=` sign please fix that

your design 
![image](https://user-images.githubusercontent.com/60612329/224185452-6155494c-4813-4e59-a1a9-57ee387ab38c.png)

required design
![image](https://user-images.githubusercontent.com/60612329/224185297-4d54288e-c0d0-4091-889a-6b1e774dec93.png)